GenderPayGap.WebUI/Views/ReportFigures/ReportFigures.cshtml

```diff
@@ -59,28 +35,26 @@
 
             <fieldset class="govuk-fieldset">
                 <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
-                    <h1 class="govuk-fieldset__heading">
-                        Enter your gender pay gap data for snapshot date @(Model.SnapshotDate.ToString("dd MMMM yyyy"))
+                    <h2 class="govuk-caption-l">Step 2 of 5</h2>
```

Comment:
Looking at the W3C standards for accessibility it was ok with with a H2 after a H1 as I'm essentially starting a new section, an argument could be made for the Step 2 of 5 being a H1 instead and the main large text being a H2 instead.

Reply:
I feel like the "Step 2 of 5" isn't a heading at all, just informative text. Probably the same with "for <organisation>". Can we make them spans instead of headings? And maybe the organisation name should be a `govuk-caption-m` instead of `heading`